addons/bestja_base/static/src/css/bestja.css

```diff
@@ -144,3 +144,7 @@ have horizontal space for one button only */
   display: none;
 }
 
+/* Hide message from inboxes when there are no messages */
```

Comment:
This description is written in a confusing way. When there are no messages, there should also be no message to hide.